lib/printer.ts

```diff
@@ -2507,7 +2494,6 @@ function genericPrintNoParens(path: any, options: any, print: any) {
     case "XMLComment":
     case "XMLProcessingInstruction":
     default:
-        debugger;
```

Comment:
Not sure if there was any particular reason this was here. I found it when I went to make sure I didn't leave any of my own debuggers in.

Reply:
This is a common place where printing can crash, because an unimplemented type is encountered. I've found it helpful to have a `debugger` statement already here, so that I can quickly debug these cases.